transformers/shared/constants/method.go

```diff
@@ -26,6 +26,7 @@ import (
 func Cat100ABI() string     { return constants.GetContractABI("MCD_CAT_1_0_0") }
 func Cat110ABI() string     { return constants.GetContractABI("MCD_CAT_1_1_0") }
 func CdpManagerABI() string { return constants.GetContractABI("CDP_MANAGER") }
+func DogABI() string        { return constants.GetContractABI("MCD_DOG_1_x_x") }
```

Comment:
We'll probably want to include the version in this method name, so that if a new Dog contract is deployed in the future, we'll already have the versioning in place.